src/main/java/net/sf/jabref/logic/undo/UndoRedoEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.sf.jabref.logic.undo;
+
+
+public class UndoRedoEvent extends AddUndoEvent {
```

Comment:
Just duplicate the code and do not introduce inheritance if it is semantically inadequate.

Maybe the meaning of "AddUndoEvent" gets clearer if it is named "AddUndoableActionEvent"?